src/cli.ts

```
@@ -568,14 +168,14 @@ export async function startup(argv: any, version?: string, tracer?: Tracer, ctx?
     }
     const lastReadOrdersTimestamp = Math.floor(Date.now() / 1000);
     const tokens = getOrdersTokens(ordersDetails);
-    options.tokens = tokens;
+    (options as any).tokens = tokens;
 
     // get config
     const config = await getConfig(
         options.rpc,
-        options.key ?? options.mnemonic,
+        (options.key ?? options.mnemonic)!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)